lib/Wrapper/AMDGPU/OptSchedDDGWrapperGCN.h

```diff
@@ -7,10 +7,10 @@
 #ifndef LLVM_OPT_SCHED_DDG_WRAPPER_GCN_H
 #define LLVM_OPT_SCHED_DDG_WRAPPER_GCN_H
 
+#include "../OptSchedDDGWrapperBasic.h"
+#include "../OptimizingScheduler.h"
```

Comment:
Don't use relative includes.

```suggestion
#include "Wrapper/OptSchedDDGWrapperBasic.h"
#include "Wrapper/OptimizingScheduler.h"
```

If we want a more unique prefix, we should move `lib/Wrapper` to `lib/opt-sched/Wrapper`.